OpenSim/Sandbox/MatlabScripts/opensimTimeSeriesTableToMatlab.m

```diff
@@ -1,12 +1,23 @@
 function data = opensimTimeSeriesTableToMatlab(table)
 
 
+%%
+import org.opensim.modeling.*
+%%
 nCol = table.getNumColumns;
 nRow = table.getNumRows;
```

Comment:
I much prefer to use `()` at the end of method names, even though MATLAB lets you omit them. Otherwise, these look like variables, not functions.